ci-operator/step-registry/hypershift/mce/power/conformance/hypershift-mce-power-conformance-workflow.yaml

```
@@ -8,8 +8,10 @@ workflow:
       - ref: ingress-aws-nlb-manifest
       - chain: ipi-aws-pre
       - ref: hypershift-mce-install
+      - ref: hypershift-mce-conf-os-images
       - ref: hypershift-mce-power-create
     test:
+      - ref: hypershift-mce-agent-info
       - chain: hypershift-mce-power-test
```

let's move after test, which ensures the cluster goes to completed state before collecting meaningful info.
```suggestion
      - chain: hypershift-mce-power-test
      - ref: hypershift-mce-agent-info
```

---

Thanks, Done